File: Решения на С/IF/If20.c
#include <stdio.h>
#include <stdlib.h>

int main()
{
    int a, b, c, d1, d2;
    scanf("%i", &a);
    scanf("%i", &b);
    scanf("%i", &c);
    d1 = abs(a - b);
    d2 = abs(a - c);
    if (d1 <= d2) {
        printf("b is nearer than c\n");
        printf("distance is equal to ");
        printf("%i", d1);
    } else {
        printf("c is nearer than b\n");
        printf("distance is equal to ");
        printf("%i", d2);
    }
}

File: Решения на Python/IF/IF20.py
a = int(input())
b = int(input())
c = int(input())

AB = abs(a - b)
AC = abs(a - c)

if AB < AC:
    print('Точка B ближе, расстояние =', b)
elif AB > AC:
    print('Точка C ближе, расстояние =', c)
else:
    print('Точки равноудалены, расстояние =', b)



